.github/workflows/on-release.yaml

```diff
@@ -51,6 +51,12 @@ jobs:
       - name: Install dependencies
         run: yarn install --frozen-lockfile
 
+      - name: Validate that i18n locale files are updated and contains 'ai' prefix in keys
+        run: |
+          chmod +x ./scripts/validatei18n_files.sh
+          ./scripts/validatei18n_files.sh
+        shell: bash
+
```

Comment:
Same comment as in on-push workflow. When the release event occurs the latest code/commit should have been tested already as part of a pull-request event.